tests/SpecTests/CommandExpectations.php

```diff
@@ -109,6 +113,7 @@ public static function fromTransactions(array $expectedEvents)
 
         $o->ignoreCommandFailed = true;
         $o->ignoreCommandSucceeded = true;
+        $o->ignoredCommandNames = ['buildInfo', 'getParameter', 'configureFailPoint'];
```

Comment:
The [transaction spec test README](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst) doesn't talk about needing to filter out APM events for the `targetedFailPoint` operation. 

I wonder if that should be clarified in either [Command Started Events](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#command-started-events) or the [`targetedFailPoint`](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#id7) section. If the latter, the operation could specify that the fail point should be set via a different client to avoid polluting the list of observed APM events for the client under test.

Granted, PHP would still need to work around this on its own because we can only listen to events from a global context, but that may still be a worthwhile spec improvement.

As for the other two command here, I assume they're needed because `configureFailPoint()` will not execute a version check each time. I'd suggest adding a comment block above this line to explain why all three of these events are ignored.

Reply:
The `buildInfo` and `getParameter` commands are because of us checking for the availability of the `configureFailPoint` command. With the introduction of the `targetedFailPoint` operation in spec tests, it may become necessary to filter these commands as they will inevitably be caught up in the command loggers enabled for spec tests. I'll create a SPEC ticket to add this to the "Command Started Events" section using similar language.

I'll also add a comment here as to why we're ignoring `buildInfo` and `getParameter` commands 👍

Reply:
Created [SPEC-1434](https://jira.mongodb.org/browse/SPEC-1434) to track this.